### PR TITLE
1081 stats

### PR DESCRIPTION
**Pull request by derekjn**
Adds global stats that live for the duration of the database. Each database's stats file is timestamped when it is first seen, which should be good enough for uniqueness.

This also eliminates the `libcurl` dependency.

### CODE REVIEW

File: src/backend/pipeline/update.c [hunk 1]
```diff
@@ -31,114 +30,204 @@
...
- * 0) Operating system
- * 1) Number of continuous views
- * 2) Aggregate input/output rows/bytes
- */
-void
-UpdateCheck(HTAB *all_dbs, bool startup)
+static int
+post(char *payload)
+{
+  struct hostent *server;
```
Comment (usmanm) [inline src/backend/pipeline/update.c]: Fix indentation.

**Comment by usmanm** [comment]
Looks good! +1 on removing `libcurl`.

File: src/backend/pipeline/update.c [hunk 2]
```diff
@@ -31,114 +30,204 @@
 #include "utils/rel.h"
 #include "utils/syscache.h"
 
+#define ANON_ENDPOINT		"localhost"
```
Comment (usmanm) [inline src/backend/pipeline/update.c]: Might want to change this to: `"http://anonymous.pipelinedb.com/check"`?

**Comment by usmanm** [comment]
Maybe we should pass `-c anonymous_update_checks=false` to `pg_regress`, Python tests and `make run`.

File: src/backend/pipeline/update.c (hunk 2 above)
Comment (derekjn) [inline src/backend/pipeline/update.c]: Haha, that is probably a good idea.

**Pull request closed without merging**